lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -1318,18 +1318,20 @@ namespace Js
                 IntlEngineInterfaceExtensionObject* intlExtensionObject = static_cast<IntlEngineInterfaceExtensionObject*>(nativeEngineInterfaceObj->GetEngineExtension(EngineInterfaceExtensionKind_Intl));
                 if (args.Info.Count == 2)
                 {
-                    auto undefined = scriptContext->GetLibrary()->GetUndefined();
-                    CallInfo toPass(callInfo.Flags, 7);
-                    return intlExtensionObject->EntryIntl_CompareString(function, toPass, undefined, pThis, pThat, undefined, undefined, undefined, undefined);
+                    // we can fast-path this call if no locale or options negotiation needs to be done in Intl.js
+                    CallInfo toPass(callInfo.Flags, 3);
```

Comment:
>callInfo.Flags, 3 [](start = 36, length = 17)

No more CallFlags_Eval? (I don't actually know what this means here, but wondering why it changed?)

Reply:
What? We never used CallFlags_Eval here...